### DISCUSSION TITLE
Input window showing black background

### DISCUSSION THREAD

**Discussion by guilhas07** [stevearc/dressing.nvim#121] [Q&A] [answered]
I was trying to remove the black background like on the README.md images but without success. It is worth to note that all my window highlight groups have bg set to none. This seems related with https://github.com/neovim/neovim/issues/18576 but I'm not sure. If anyone knows if there is a way to fix this please let me know!
![image](https://github.com/stevearc/dressing.nvim/assets/48023091/dac89a19-8315-4c42-92a0-cfba029e674f)

**Comment by stevearc** [comment, marked as answer]
It is most likely related to that issue. Try 
```lua
require("dressing").setup({
  input = {
    win_options = {
      -- Window transparency (0-100)
      winblend = 0,
    },
  }
})
```

**Reply by guilhas07** [reply]
Ty! It removes the transparency, but looks much better. Maybe this should be the default while the issue isn't fixed, and transparency can't be "fully achieved". I would be up to create the PR if that is something you think would be nice to modify.

**Reply by stevearc** [reply]
Yeah that sounds reasonable. It was an easy change so I went ahead and did it myself
